tmt/utils/__init__.py

```diff
@@ -5611,7 +5611,8 @@ def normalize_string_dict(
 
     if isinstance(raw_value, dict):
         return {
-            str(key).strip(): str(value).strip() for key, value in raw_value.items()
+            str(key).strip(): (value if isinstance(value, (list, dict, tuple))
+                               else str(value).strip()) for key, value in raw_value.items()
```

Comment:
Why this change? Well, why the changes to `normalize_dict_string` and kickstart fields?

Reply:
I see, I get it now. But that's now a bit of a problem with regard to the name of this function, right? It's no longer "string dict" as values are no longer just strings. It looks like we should have more specialized version of this function, to normalize kickstart, rather than mangling the existing one.

Reply:
Yeah, I know we should rename the function,and I was waiting for your comment^^, just to be clear, you think we should have a new function,named normalize_kickstart,right?^^

Reply:
Yep, `normalize_kickstart` now looks more fitting the situation.

Reply:
Updated^^
https://github.com/teemtee/tmt/pull/3354/commits/e01d3c78d65d25b842ca275d2e2b0766c0dbf80a